_posts/2023-01-26-rust-32.0.0.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: "January 2023 Rust Apache Arrow Highlights"
+date: "2023-01-26 00:00:00 +0000"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Introduction
+
+With the recent release of [32.0.0](https://crates.io/crates/arrow/32.0.0) of the Rust implementation of [Apache Arrow](https://arrow.apache.org/), it seemed timely to highlight some of the communities work since the [last update](https://arrow.apache.org/blog/2022/06/16/rust-16.0.0/).
```

Comment:
Maybe?

```suggestion
With the recent release of [32.0.0](https://crates.io/crates/arrow/32.0.0) of the Rust implementation of [Apache Arrow](https://arrow.apache.org/), it seemed timely to highlight some of the community works since the [last update](https://arrow.apache.org/blog/2022/06/16/rust-16.0.0/).
```